cpu/native/periph/timer.c

```diff
@@ -27,6 +27,8 @@
 #include <mach/mach_init.h>
 #include <mach/mach_port.h>
 #include <mach/mach_host.h>
+/* Both OS X and RIOT typedef thread_t. timer.c does not use either thread_t. */
+#define thread_t riot_thread_t
```

Comment:
Shouldn't this rather be `mac_thread_t`? Yes, this file doesn't use either, but at least the type shoult point to the right implementation ;-).